examples/hello-world/src/index.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import {IntlProvider} from 'react-intl';
+import {IntlProvider,addLocaleData} from 'react-intl';
 import App from './App';
-
+import fr from 'react-intl/locale-data/fr';
+addLocaleData(fr);
```

Comment:
I want to avoid using more than one language for the hello-world example. React Intl is still useful even if you're only using English.